CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+## Getting started
+Before you begin:
+- Apocryph repositories use [C# (.NET 5.0)](https://dotnet.microsoft.com/download) and [Solidity](https://ethereum.org/en/developers/local-environment/);
+- Check the README / documentation of the repository to which you are interested in contributing;
+- Setup your local environment according to the README / documentation.
+
+## How to contribute?
+When contributing to this repository, please first pick an issue marked as "open to contributors" or initiate a discussion using a new issue. You can check the suitable issues for the repository or check the Apocryph GitHub Project for overview of all open issues across all repositories.
+
+### What are issues open to contributors?
+All of the work in the project is organized using milestones and issues. Certain issues are marked as "open to contributors". These issues are specifically curated by the maintainers as suitable for contributors. 
```

Comment:
Question left unanswered: what happens when someone opens a pull request either without a backing issue or for an issue that is not open for contribution? What steps would they have to take at that point to claim a reward for such a PR, if it is decided that it is a good change.